spec/tic_tac_toe/board_spec.clj

```diff
@@ -0,0 +1,7 @@
+(ns tic-tac-toe.board-spec
+  (:require [speclj.core :refer :all]
+            [tic-tac-toe.board :refer :all]))
+
+(describe "a board"
+  (it "returns an empty board"
+    (should= '(- - - - - - - -) (empty-board) )))
```

Comment:
you may need to give board an alias.. (my clojure is a little rusty so it'll be worth checking the various ways to import in clojure)